packages/pharos/package.json

```diff
@@ -84,7 +84,28 @@
     "sinon": "^12.0.1",
     "style-dictionary": "^3.0.1",
     "ts-lit-plugin": "^1.2.1",
-    "typescript": "^4.5.2"
+    "typescript": "^4.9.4"
   },
-  "customElements": "custom-elements.json"
+  "customElements": "custom-elements.json",
+  "exports": {
+    ".": "./lib/index.js",
+    "./lib": "./lib/index.js",
+    "./lib/index": "./lib/index.js",
+    "./lib/react-components": "./lib/react-components/index.js",
+    "./lib/styles/pharos": "./lib/styles/pharos.scss",
+    "./lib/styles/typography": "./lib/styles/pharos.scss",
+    "./lib/styles/typography.scss": "./lib/styles/typography.scss",
+    "./lib/styles/fonts.css": "./lib/styles/fonts.css",
+    "./lib/styles/variables.css": "./lib/styles/variables.css",
+    "./lib/styles/variables": "./lib/styles/variables.js",
+    "./lib/styles/pharos-alert.css": "./lib/styles/pharos-alert.css",
+    "./lib/styles/pharos-footer.css": "./lib/styles/pharos-footer.css",
+    "./lib/styles/pharos-modal.css": "./lib/styles/pharos-modal.css",
```

Comment:
I'd bump @NazimHAli's suggestion about using patterns for these, as not remembering to do this will likely result in some patch fixes in the future. Unless there was a reason that didn't work?

Reply:
so this seems to work
```
    "./lib/styles/*": "./lib/styles/*.js",
    "./lib/styles/*.css": "./lib/styles/*.css",
    "./lib/styles/*.scss": "./lib/styles/*.scss",
    "./lib/styles/pharos": "./lib/styles/pharos.scss",
  ```
  i needed to export styles/pharos.scss separately because storybook was complaining it couldn't find it with `@use '@ithaka/pharos/lib/styles/pharos';`